deployments/multus-daemonset-thick.yml

```diff
@@ -198,6 +202,9 @@ spec:
         - name: cnibin
           hostPath:
             path: /opt/cni/bin
+        - name: hostdir
```

Comment:
maybe a more specific name, `hostrootdir` or something, seems kind of overloaded between `hostPath` and `hostdir`

Reply:
Thanks. Changed to hostroot.